doc/theory/glossary.tex

```diff
@@ -287,6 +287,22 @@
 	
 	The name is inspired from the case when the persistence diagram comes from persistent homology.
 	
+	\subsection*{Heat kernel:} \label{heat kernel}
+	
+	Given a set $\xi$, a kernel $k:\xi \times \xi \rightarrow \mathbb{R}$ is a kernel, if there exists a Hilbert space $H$ with inner product $\langle \cdot, \cdot \rangle$ and a function $\phi: \xi \rightarrow H$, such that $k(x,y) = \langle \phi(x), \phi(y) \rangle$.
+	
+	In contrast to vectorisation methods, kernel methods do not give an explicit feature map nor a vector representation. Nevertheless,such methods remain popular for tasks like classification.
```

Comment:
Is there a strong enough difference between "feature map" and "vector representation" that both have to be included?

Reply:
Good point,I removed the latter.